src/components/Blog/BlogArticlesBoard/BlogArticlesBoard.tsx

```diff
@@ -1,112 +1,223 @@
+const pageSize = 6;
 export const BlogArticlesBoard = ({
   data,
   url,
   tags,
 }: BlogArticlesBoardProps) => {
   const theme = useTheme();
   const { t } = useTranslation();
+  const [tabId, setTabId] = useState<number | undefined>(0);
   const [openDropdown, setOpenDropdown] = useState<boolean>(false);
-  const [categoryId, setCategoryId] = useState<number | undefined>(undefined);
   const [catLabel, setCatLabel] = useState<string | undefined>(
     t('blog.allCategories'),
   );
+
+  console.log('####tags', tags);
   const isDesktop = useMediaQuery((theme: Theme) => theme.breakpoints.up('md'));
   const handleTagsClick = useCallback(
     (id: number, label?: string) => () => {
       if (!isDesktop && !openDropdown) {
+        console.log('CHECK HANDLER OPEN', id, label);
         setOpenDropdown(true);
       } else {
+        console.log('CHECK HANDLER CLOSE', id, label);
         setOpenDropdown(false);
       }
-      setCategoryId(id);
+      setTabId(id);
       setCatLabel(label);
     },
     [isDesktop, openDropdown],
   );
 
   const filteredTags = useMemo<TabProps[]>(() => {
     // default "All" Category
-    const defaultFilter = {
-      id: 0,
-      label:
-        !isDesktop && categoryId !== 0 ? catLabel : t('blog.allCategories'),
-      icon: !isDesktop && (
-        <ArrowDropDownIcon
-          sx={{
-            position: 'absolute',
-            right: theme.spacing(1),
-            marginBottom: 0,
-          }}
-        />
-      ),
-      attributes: [],
-      value: 0,
-      onClick: handleTagsClick(0, t('blog.allCategories')),
-      disabled: false,
-    };
+    // const defaultFilter = {
+    //   id: 0,
+    //   label:
+    //     !isDesktop && tabId !== 0 ? catLabel || '' : t('blog.allCategories'),
+    //   icon: !isDesktop && (
+    //     <ArrowDropDownIcon
+    //       sx={{
+    //         position: 'absolute',
+    //         right: theme.spacing(1),
+    //         marginBottom: 0,
+    //       }}
+    //     />
+    //   ),
+    //   attributes: [],
+    //   value: 0,
+    //   onClick: handleTagsClick(0, t('blog.allCategories')),
+    //   disabled: false,
+    // };
     // tags
-    const output = tags?.map((el, index: number) => {
+    const output = tags?.data.map((el, index: number) => {
       return {
-        label: el.attributes.Title || undefined, //el.attributes.Title,
+        label: el.attributes.Title || '',
         value: el.id,
-        onClick: handleTagsClick(el.id, el.attributes.Title),
+        onClick: handleTagsClick(index, el.attributes.Title),
         // disabled: false,
       };
     });
     // merge default + tags
-    output?.unshift(defaultFilter);
+    // output?.unshift(defaultFilter);
     return output;
-  }, [categoryId, catLabel, handleTagsClick, isDesktop, t, tags, theme]);
+  }, [handleTagsClick, tags?.data]);
+
+  const handleClick = (event: { target: HTMLElement }) => {
+    console.log('event', event);
+    const classList = event.target.classList;
```

Comment:
All "Board" related files are the version that allows navigating between categories via Tabs.
In this new "Collection" component, we just show all categories in its own section with no need to use tabs.

The board seems much nice, but we want to please SEO and try it for now. But possibly coming back, so this is the "duplicate" you are referring to - I suggested A/B testing both against each other

Reply:
This component does not seem to be used anywhere, I don't understand the last point but checking the classList of the component is a bad practice and not resilient (because we are in a reactive environment, you should have that information already without doing some dom checking [which can also not be updated at the time you are checking it])

Reply:
Oh, I wanted to make sure a certain component is being clicked. But I can see that this might not be the best approach.. I´d love to understand how this can be improved!

I used this same approach in other components such as the WelcomeScreen.tsx!